src/app/enrichment-models/enrichment-models.service.ts

```diff
@@ -1,28 +1,27 @@
 import { Injectable } from '@angular/core';
-import { Http } from '@angular/http';
+import { HttpClient } from '@angular/common/http';
+// tslint:disable-next-line:import-blacklist
 import { Observable } from 'rxjs';
 
 import { ConfigService } from '../config/config.service';
 import { EnrichmentModels } from './enrichment-models';
 
-
-
 @Injectable()
 export class EnrichmentModelsService {
-  private enrichmentModelsUrl = 'enrichment/models';
+  private readonly enrichmentModelsUrl = 'enrichment/models';
 
   constructor(
-    private http: Http,
+    private http: HttpClient,
     private config: ConfigService
-  ) { }
+  ) {}
 
   getBackgroundModels(datasetId: String): Observable<EnrichmentModels> {
-    const url = `${this.enrichmentModelsUrl}/${datasetId}`;
+    const url = `${this.config.baseUrl}${this.enrichmentModelsUrl}/${datasetId}`;
```

Comment:
We should choose a single way of producing the URL strings - either concatenating with `+` or formatting.

Reply:
I'm resolving these conversations as they have been moved to a different card in our Trello.